node/validation/manifest/schema.ts

```diff
@@ -24,6 +24,22 @@ const routesSchema = {
   additionalProperties: false,
 }
 
+const functionConfigSchema = {
+  type: 'object',
+  required: ['excluded_patterns'],
```

Comment:
Why is it required?

Reply:
🤷 I don't remember. Removed it in https://github.com/netlify/edge-bundler/pull/274/commits/0fa20a9faee3ed274554d47db8cf298c2f8bbd3a.